src/KingSystem/System/Hio.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <heap/seadDisposer.h>
+#include <hostio/seadHostIONode.h>
+#include "KingSystem/Utils/Types.h"
+
+namespace ksys {
+
+class Hio final : public sead::hostio::Node {
+    SEAD_SINGLETON_DISPOSER(Hio)
+
+public:
+    Hio() = default;
+    ~Hio() = default;
+};
+KSYS_CHECK_SIZE_NX150(Hio, 0x28);
+
+}  //namespace ksys
```

Comment:
```suggestion
}  // namespace ksys
```
(this will fix the lint failure)